pos_multi_session/static/src/js/pos_multi_session.js

```diff
@@ -90,9 +90,14 @@ odoo.define('pos_multi_session', function(require){
                     self.get_bus('sync_server').add_channel_callback(channel_name, callback, self);
                     self.sync_bus = self.get_bus('sync_server');
                     self.get_bus('sync_server').start();
+                    if (!self.config.autostart_longpolling) {
+                        self.bus.activated = false;
+                        self.bus.start();
+                    }
                 } else {
                     self.sync_bus = self.get_bus();
                     if (!self.config.autostart_longpolling) {
+                        self.sync_bus.activated = false;
```

Comment:
autostart_longpolling is only for main server!

Reply:
@yelizariev Yes, that is. self.get_bus(); returns pos.bus - main server

Reply:
ok